src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -209,6 +209,10 @@
     <value>Automatically copy selection to clipboard</value>
     <comment>Header for a control to toggle whether selected text should be copied to the clipboard automatically, or not.</comment>
   </data>
+  <data name="Globals_TrimBlockSelection.Header" xml:space="preserve">
+    <value>Remove trailing white-spaces when copying text in rectangular selection to clipboard</value>

```

Comment:
I'm ok with this. But @cinnamon-msft might want to take a look here haha

Reply:
@carlos-zamora how do you feel about this

```suggestion
    <value>Remove trailing white-space in rectangular selection</value>
```

I am okay with the negative boolean. It _enables a destructive operation_.

Reply:
I like that a lot more. This is one of those cases where I'd prefer it to be short but ambiguous. Thanks!